### PR TITLE
Add calibration level definition

### PR DESCRIPTION
**Pull request by lmichel**
This vocabulary is based on the calibration levels as defined in Obcore1.1.

- Its purpose is to replace the MANGO literal enumeration for the calibration level of physical properties.
- Calibration-level vocabulary has one root which is specific for Obscore.
- It might be extended whith levels defined in other standards (EPN-TAP2 CODMAC PSA  NASA  PDS). 
- The equivalences between these are detailed in EPN-AP 2.0 section 2.1.2.

### CODE REVIEW

**Review by msdemlei — CHANGES_REQUESTED** [review]
Looks good to me; I think I'd be happy to squash-merge.  To minor points, though:

(1) in the README, EPN-AP needs to be EPN-TAP

(2) I'm not too happy with "obscore" as the identifier; I'd prefer "obscore-level" or so.  More importantly, the label of that concept should be in singular: "Obscore calibration level"; that way, building an is-a sentence will work out right ('Raw' is an 'Obscore calibration level', not level*s*).

Thanks!

File: calibration-level/calibration-level.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+obscore-level;1;Obscore calibration level;Obscore 1.1 calibration level;
+obscore-raw;2;Raw instrumental data;(0) Raw instrumental data, in a proprietary or internal data provider defined format, that needs instrument specific tools to be handled.;
+obscore-instrumetal;2;Instrumental data;(1) Instrumental  data  in  a  standard  format  (FITS,  VOTable,  SDFITS,  ASDM,  etc.) which could be manipulated with standard astronomical packages.;
```
Comment (BaptisteCecconi) [inline calibration-level/calibration-level.csv]: please fix typo in term: `obscore-instrumetal` => `obscore-instrumental`

**Comment by BaptisteCecconi** [comment]
I'm happy with having a vocabulary for this series of terms. 

I would rather name it "processing-level", since the `obscore-enhanced` and `obscore-analysis` are similar to the 'reprojected' and 'derived' data processing levels that can be found elsewhere. 

My main question is about having an `obscore-level` main concept/class. Does this mean that we will have other roots (like those mentioned in the readme?) and link them within the vocabulary?

I'll update the vocab.conf file and approve with preliminary status.

**Review by BaptisteCecconi — CHANGES_REQUESTED** [review]

**Comment by lmichel** [comment]
- `calibration-level `has been renamed as `processing-level`
- Typos corrected
- README updated accordingly

**Comment by lmichel** [comment]
> My main question is about having an obscore-level main concept/class. Does this mean that we will have other roots (like those mentioned in the readme?) and link them within the vocabulary?

Yes, we might consider having other roots since processing levels are used in different contexts. 
The idea is to set then cross links between different words having the same meaning.
This does not prevent to merge at some point  these different words into a common standard.

**Review by BaptisteCecconi — APPROVED** [review]

**Review by BaptisteCecconi — APPROVED** [review]

**Review by msdemlei — CHANGES_REQUESTED** [review]
I'd say we ought to have the necessary vocabs.conf changes in this commit, too.

This opens the interesting question whether this should be a property or a class vocabulary.  This is operationally unimportant, but getting it right now will make later RDF work simpler.

Without having thought deeply about it, I think it's class.  The triples you'd build with it would probably look like

```
  (http://example.org/dataset, *has-been-processed-to, raw)
```

We don't have that property (and won't need it for now), but it's good that we *could* define it.

Against that,

```
  (http://example.org/dataset, *has-been-processed-to, ???)
```

doesn't feel right, and I can't see plausible scenarios that would profit from being able to have objects here.

**Comment by BaptisteCecconi** [comment]
I agree it's a should be a class vocabulary. I already updated the `vocabs.conf` file, so we can merge I think

**Review by BaptisteCecconi — APPROVED** [review]

**Pull request merged**